.circleci/config.yml

```diff
@@ -654,26 +736,30 @@ orbs:
                 else
                   cat ${WORKSPACE}.log | grep -E 'aws_ecs_(service|cluster)|aws_s3.*|aws_rds.*|aws_db.*|aws_lb.*' | grep -E 'destroyed|replaced'
                 fi
-
-                if [[ "$?" -eq 0 ]]
+                  if [[ "$?" -eq 0 ]]
                 then
                    echo terraform log scan found a possible destructive change.
                    echo please review before approval.
-
-                   ~/project/scripts/pipeline/auto_approve_hold/slack_approval_warning.sh \
-                                        "${SLACK_DEV_WEBHOOK}" \
-                                        "${CIRCLE_PULL_REQUEST}" \
-                                        "${CIRCLE_JOB}" \
-                                        "${CIRCLE_BUILD_URL}" \
-                                        "${approval_step}"
+                   exit 1
                 else
-                  echo terraform log scan found no destructive changes. auto-approval will take place.
-                  ~/project/scripts/pipeline/auto_approve_hold/auto_approve_hold.sh \
-                                        "${CIRCLE_WORKFLOW_ID}" \
-                                        "${approval_step}" \
-                                        "${CIRCLECI_API_KEY}"
+                    echo terraform log scan found no destructive changes. auto-approval will take place.
+                    exit 0
                 fi
 
+          - my_slack/notify:
+              when: on_fail
+              slack_channel: C01BDFQHTRU
```

Comment:
Should the slack channel id be a env?

Reply:
perhaps this is a good idea. i'll look into it now @johnnolan. would make the channel id a secret as a project variable.